packages/subgraph/src/plugin/plugin.ts

```diff
@@ -1,24 +1,221 @@
-import {DaoPlugin} from '../../generated/schema';
-import {NumberStored} from '../../generated/templates/Plugin/Plugin';
-import {generatePluginInstallationEntityId} from '@aragon/osx-commons-subgraph';
-import {Address, dataSource} from '@graphprotocol/graph-ts';
+import {
+  Action,
+  DaoPlugin,
+  MultisigProposal,
+  MultisigApprover,
+  MultisigProposalApprover,
+} from '../../generated/schema';
+import {
+  ProposalCreated,
+  ProposalExecuted,
+  MembersAdded,
+  MembersRemoved,
+  Plugin,
+  Approved,
+  MultisigSettingsUpdated,
+} from '../../generated/templates/Plugin/Plugin';
+import {
+  generateActionEntityId,
+  generateEntityIdFromAddress,
+  generatePluginEntityId,
+  generateProposalEntityId,
+} from '@aragon/osx-commons-subgraph';
+import {Address, dataSource, store} from '@graphprotocol/graph-ts';
 
-export function handleNumberStored(event: NumberStored): void {
+export function generateVoterEntityId(
+  memberEntityId: string,
+  proposalId: string
+): string {
+  return [memberEntityId, proposalId].join('_');
+}
+
+export function generateMemberEntityId(
```

Comment:
this could be moved to an util or helper file

Reply:
Moved 499ca486790012f3a540ba3e0a0fe6b30f4e9357